### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by jnfernQA**
Added Feature might have added wrong

### CODE REVIEW

**Review by QA-ArsalanAsad — COMMENTED** [review]
its nice

**Review by QA-ArsalanAsad — COMMENTED** [review]
good

**Review by QA-ArsalanAsad — APPROVED** [review]
,,

File: TextFeature1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello Team Scorpio
```
Comment (MuratOzturk18) [inline TextFeature1.txt:1]: Cool Cool

**Review by MuratOzturk18 — APPROVED** [review]

**Review by aktarshahnazqa — APPROVED** [review]

**Comment by QA-ArsalanAsad** [comment]
finished

**Pull request merged**